### DISCUSSION TITLE
Failed to connect to bitbucket.org port 443 after 21310 ms: Timed out

### DISCUSSION THREAD

**Discussion by ManiverReddy** [community/community#38051] [Actions]
Hai All, 

I am trying to retrieve a repository from bitbucket using Github desktop but i am receiving this error. I tried to uninstall and re install github and tried cloning the repo from start. Still the issue is same. Would be glad if i could get some insights on this.

Error :- Failed to connect to bitbucket.org port 443 after 21310 ms: Timed out

Thanks in advance.

**Comment by david-engelmann** [comment]
Do you have the proper Bitbucket authentication files or flow setup? If you are trying this through a github actions runner, it may be difficult to submit the app password when BitBucket prompts you to enter it. May work better with ssh keys!

**Reply by ManiverReddy** [reply]
Hello David, I have proper credentials of bitbucket and used the same on github desktop. I am using PAT Personal Access Token as password while logging in. It worked well, 2 days ago. But its throwing this error recently.
